src/main/java/racingcar/step4/domain/Winners.java

```diff
@@ -0,0 +1,31 @@
+package racingcar.step4.domain;
+
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+public class Winners {
+
+  private final List<Car> winners;
+
+  private Winners(List<Car> winners) {
+    this.winners = winners;
+  }
+
+  public static Winners pickWinners(List<Car> candidates) {
+    Car winnerCandidate = Collections.max(candidates, Comparator.comparing(it -> it.getPosition().get()));
```

Comment:
`Position` 을 값객체로써 원시값을 포장했으니, `Comparable` 을 확장해서 인스턴스비교로 활용해보시는건 어떠신가요?

`Car` 도 마찬가지 `Comparable`로 `Position` 을 비교하게 만들면

 ```suggestion
    Car winnerCandidate = Collections.max(candidates);
```

로 변경할 수 있을것 같아요 :)